xml/containers_basics.xml

```diff
@@ -1,78 +1,202 @@
+    </para>
+    <para>
+      You can use a base image as it is for running containers, but the main purpose of base images is to serve as foundations for creating custom images that can run containers with specific applications, servers, services, and so on.
+    </para>
+    <para>
+      Both base and custom images are usually available through a repository of images called registry. By default, when you fetch an image using the Docker client, you are actually pulling it from the Docker Hub registry.
+    </para>
+    <para>
+      After you pull a base image, you can use it as a starting point for building a custom image. For example, you can use the SLE base image to generate an image that is configured to run a specific application. To build a custom image, you need to create a special file called Dockerfile, containing building instructions. For example, a Dockerfile can contain instructions to update the system software, install the desired application, open specific network ports, run commands, etc.
+    </para>
+    <para>
+      Note that you can build images not only from base images, but also on top of custom images. So you can have an image consisting of multiple layers:
+    </para>
+    <para>
+      Base image -> Custom image 1 -> Custom image 2 -> Custom image 3 (
+      <!--TODO Replace with figure -->
+    </para>
+    <sect2 xml:id="sec-container-example">
+      <title>Practical Example</title>
+      <para>
+        The following procedure shows how to build a custom &docker; image that
+        can be used to run a container with a simple PHP application called
+        <literal>example</literal>
+        served using the built-in PHP development
+        server.
+      </para>
+      <procedure>
+        <title>Building an image and running container</title>
+        <step>
+          <para>
+            Install Docker:
+          </para>
+<screen>sudo zypper in docker docker-compose</screen>
+        </step>
+        <step>
+          <para>
+            Enable or start Docker:
+          </para>
+<screen>sudo systemctl enable docker
+sudo systemctl start docker</screen>
+        </step>
+        <step>
+          <para>
+            To run Docker as a regular user, add the user to the
+            <literal>docker</literal>
+            group:
+          </para>
+<screen>sudo usermod -G docker -a $USER</screen>
+          <para>
+            Log out, log in again.
+          </para>
+        </step>
+        <step>
+          <para>
+            Switch to the PHP project's directory and create a file named
+            <filename>Dockerfile</filename>
+            :
+          </para>
+<screen>cd example
+touch Dockerfile</screen>
+        </step>
+        <step>
+          <para>
+          </para>
+        </step>
+        <step>
+          <para>
+            Open the _Dockerfile_ file for editing, and add the following:
+          </para>
+<screen>
+FROM php:7.4-cli
```

Comment:
It should be noted that this image would not receive support from us.

Reply:
Hm, I think it's pretty obvious, considering that we are building an example image. :smile:

Reply:
Obvious to you maybe, but I fully expect that someone will file a ticket about "php in this container is segfaulting" and expect us to investigate...

Reply:
Hahaha, you are right.